fied/nei/nei_EF_calculations.py

```diff
@@ -52,6 +53,8 @@ def __init__(self):
 
         self._data_source = 'NEI'
 
+        self.year = 2017
```

Comment:
So, it looks like there aren't any differences between the 2017 and 2020 aside from the zipped csvs. I wonder if we can delete this year instance variable and instead add a year argument to the `load_nei_data` method? 
Maybe a more future-proof approach would be to initialize the NEI method with a year argument, i.e., 
```
def __init__(self, year):
    self._year = year
```
and then then this can be accessed by `load_nei_data` and any other methods that might need to be changed for additional NEI years.